WeakAuras/Conditions.lua

```diff
@@ -819,7 +819,7 @@ function Private.UnregisterForGlobalConditions(uid)
     if next(condFuncs) == nil then
       local unitEvent, unit = event:match("([^:]+):([^:]+)")
       if unitEvent and unit then
-        dynamicConditionsFrame.units[unit]:UnregisterEvent(unitEvent)
+        pcall(dynamicConditionsFrame.UnregisterEvent.units[unit], dynamicConditionsFrame.units[unit], unitEvent, unit);
```

Comment:
That doesn't look correct:
dynamicConditionsFrame.UnregisterEvent.units[unit]

Also the unit parameter should not be necessary.